src/pages/Topic.jsx

```diff
@@ -0,0 +1,18 @@
+import { useParams } from "react-router-dom";
+import Articles from "../components/Articles";
+import Header from "../components/Header";
+import Topics from "../components/Topics";
+
+const Topic = () => {
+  const { topic } = useParams();
+
+  return (
+    <>
+      <Header />
+      <Topics />
```

Comment:
If Header and Topics is in topic.jsx and home.jsx then surely it needs to be moved higher?